tasks/python.yml

```diff
@@ -10,10 +10,6 @@
       - python-ipython-console
     state: present
 
-- name: Install Python 3.4
-  package: name=python34 state=present
-  when: common_python3_enabled
-
 - name: Install Python 3.6
```

Comment:
Upgrade?

Reply:
As far as I know this is still used for Jenkins deploy. We should address removal of Python3.6 or upgrade separately from this commit. Database deploy and Kubernetes node deploys don't really use system Python any more.